selenium/scripts/check_endpoints.pl

```diff
@@ -0,0 +1,106 @@
+#!/usr/local/bin/perl

```

Comment:
I believe the recommended way of invoking the interpreter these days is "/usr/bin/env perl".

Reply:
With plenv it is imperative that we don't accidentally invoke the default system Perl.

Reply:
done